control_mode:
    j_wheel_1: 2
    j_wheel_2: 2
    j_wheel_3: 2
    j_wheel_4: 2

fixed_joints:
    j_wheel_1: 0.0
    j_wheel_2: 0.0
    j_wheel_3: 0.0
    j_wheel_4: 0.0
    ankle_yaw_1: 0.785
    ankle_yaw_2: -0.785
    ankle_yaw_3: -0.785
    ankle_yaw_4: 0.785

define:
    - &low_stiff 200.
    - &low_damp 10.
    - &high_stiff 8000.
    - &high_damp 40.

stiffness:
    ankle_pitch_1: *low_stiff
    ankle_pitch_2: *low_stiff
    ankle_pitch_3: *low_stiff
    ankle_pitch_4: *low_stiff
    435_head_joint: *low_stiff
    hip_pitch_1: *low_stiff
    hip_pitch_2: *low_stiff
    hip_pitch_3: *low_stiff
    hip_pitch_4: *low_stiff
    hip_yaw_1: *low_stiff
    hip_yaw_2: *low_stiff
    hip_yaw_3: *low_stiff
    hip_yaw_4: *low_stiff
    j_arm1_1: *low_stiff
    j_arm1_2: *low_stiff
    j_arm1_3: *low_stiff
    j_arm1_4: *low_stiff
    j_arm1_5: *low_stiff
    j_arm1_6: *low_stiff
    j_arm2_1: *low_stiff
    j_arm2_2: *low_stiff
    j_arm2_3: *low_stiff
    j_arm2_4: *low_stiff
    j_arm2_5: *low_stiff
    j_arm2_6: *low_stiff
    knee_pitch_1: *low_stiff
    knee_pitch_2: *low_stiff
    knee_pitch_3: *low_stiff
    knee_pitch_4: *low_stiff
    torso_yaw: *low_stiff


damping:
    ankle_pitch_1: *low_damp
    ankle_pitch_2: *low_damp
    ankle_pitch_3: *low_damp
    ankle_pitch_4: *low_damp
    435_head_joint: *low_damp
    hip_pitch_1: *low_damp
    hip_pitch_2: *low_damp
    hip_pitch_3: *low_damp
    hip_pitch_4: *low_damp
    hip_yaw_1: *low_damp
    hip_yaw_2: *low_damp
    hip_yaw_3: *low_damp
    hip_yaw_4: *low_damp
    j_arm1_1: *low_damp
    j_arm1_2: *low_damp
    j_arm1_3: *low_damp
    j_arm1_4: *low_damp
    j_arm1_5: *low_damp
    j_arm1_6: *low_damp
    j_arm2_1: *low_damp
    j_arm2_2: *low_damp
    j_arm2_3: *low_damp
    j_arm2_4: *low_damp
    j_arm2_5: *low_damp
    j_arm2_6: *low_damp
    knee_pitch_1: *low_damp
    knee_pitch_2: *low_damp
    knee_pitch_3: *low_damp
    knee_pitch_4: *low_damp
    torso_yaw: *low_damp

